autopilot/scanner.go

```diff
@@ -196,12 +197,10 @@ func (s *scanner) launchHostScans() chan scanReq {
 		var offset int
 		var exhausted bool
 		for !s.isStopped() && !exhausted {
-			s.logger.Debugf("scanning hosts %d-%d", offset, offset+int(s.scanBatchSize))
-
 			// fetch next batch
-			hosts, err := s.bus.Hosts(offset, int(s.scanBatchSize))
+			hosts, err := s.bus.HostsForScanning(time.Now().Add(-s.scanMinInterval), offset, int(s.scanBatchSize))
```

Comment:
I think `maxLastScan` needs to be fixed for a single scan for `offset` and `limit` to work correctly?